Observatory-Control-System/BLOCKS/block-jira-project-workflow.rst

```diff
@@ -0,0 +1,245 @@
+--------------------------
+In the JIRA issues, there are only two formal roles defined: Proposer and Assignee. 
+
+**Proposer**
+^^^^^^^^^^^^
+The Proposer is the stakeholder requesting the observing task. 
+They are chiefly responsible for the initial ticket submission, including providing links to supporting materials, and the final ticket sign-off (see Transitioning to Done). 
+They are expected to work closely with the Assignee during the Preparation phase, but are not expected to be involved in planning when a task will take place or during the task execution (unless otherwise required). 
+Anyone can take the role of Proposer as long as they are knowledgeable in the task requirements and completion criteria. 
+
+**Assignee**
+^^^^^^^^^^^^
+The Assignee is the member of the Commissioning Science team expected to oversee the task during the preparation and execution phase. 
+They are expected to collaborate with the Proposer during task preparation, and are directly responsible for ensuring the task is Ready for execution at the summit. 
+They are expected to track task progress and advocate for task completion during the task planning meetings when necessary. 
+
+In addition, there is the role of planning the tasks for execution once they have been set to READY. 
+This is expected to be carried out in accordance with the priority assigned to the task by the Commissioning Science team. 
+While awaiting execution, tickets will remain in the READY state and their priority may be updated as needed.
+Ticket Assignees are expected to track the status of execution and advocate for planning of tasks as needed. 
+
+And finally, there is the role that is filled by the person responsible for task execution at the summit, in most cases an Observing Specialist. 
+Although responsible for execution of the task (arguably the most import step), this person does not have any defined responsibilities in the JIRA Workflow itself. 
+However, they do hold the final say of whether a task can be executed on the summit (as a final evaluation is implicit in their role), and have full discretion to transition tickets back to Preparation if they are found to be lacking instruction. 
+
+Overview of Workflow
+--------------------
+Here we provide a bulleted list illustrating an example workflow. 
+
+* Stakeholder requires an observing task to be completed at the summit, and describes the task in an LVV ticket, SITCOM ticket, or Confluence page. 
+* Stakeholder proposes a new BLOCK Jira ticket, providing links to supporting material and assuming responsibility of Proposer. 
+* New BLOCK Jira ticket is reviewed by Commissioning Science team during one of the regular, weekly planning meetings, and accepted/rejected for scheduling. 
+* New BLOCK Jira ticket is given a priority for scheduling and a Commissioning Scientist is Assigned the ticket, taking on the responsibility of Assignee. 
+* Proposer and Assignee work together to prepare BLOCK ticket for execution.
```

Comment:
Maybe you can include one extra step to tell that before marking as Ready there will be a validation phase done by a Com Sci.